remix-lib/src/execution/execution-context.js

```diff
@@ -2,9 +2,9 @@
 'use strict'
 var Web3 = require('web3')
 var EventManager = require('../eventManager')
-var EthJSVM = require('ethereumjs-vm')
+var EthJSVM = require('ethereumjs-vm').default
```

Comment:
If possible use `const { default as EthJSVM } = require('ethereumjs-vm')`. If ever another project is using this project with webpack or any modern bundler it can take advantage of tree-shaking (just bundle the default, and not everything else).

Reply:
Actually `const { default as EthJSVM } = require('ethereumjs-vm')` doesn't work.

Reply:
ah yes, this isn't default in node...